resources/lang/pl/home.php

```diff
@@ -106,13 +106,15 @@
         ],
         'header' => [
             'welcome' => 'Witaj, <strong>:username</strong>!',
-            'messages' => 'Masz 1 nową wiadomość|Masz :count nowych wiadomości',
+            'messages' => 'Masz 1 nową wiadomość|Masz :count nowe wiadomości|Masz :count nowych wiadomości',
```

Comment:
a nie czasem Masz :count nowych wiadmości|Masz :count nowe wiadomości|Masz :count nowych wiadomości, bo na stronie cały czas przez to pisze masz 1 nową wiadomość